packages/user-interface/src/components/FileUpload.tsx

```diff
@@ -9,20 +10,23 @@ export interface UploadedFile {
 }
 
 interface FileUploadProps {
+  context?: object;
```

Comment:
```suggestion
  context: object;
```

if it's an empty object is can't be undefined right?

Reply:
This component can be used elsewhere though where formio component context is not available

Reply:
Oke fair. Now it's only used in this situation right?

Reply:
Yea, I guess we can make it optional when and if we need to reuse the fileupload component. since we also probably need to make the `uploadLink` property part of the props in such a case